src/m365/tenant/commands/people/profileCardProperties.ts

```diff
@@ -0,0 +1,23 @@
+export const profileCardPropertyNames: string[] = [
+  'userPrincipalName',
+  'faxNumber',
+  'streetAddress',
+  'postalCode',
+  'state',
+  'mailNickname',
```

Comment:
No that's incorrect @jwaegebaert, we've discussed this in basecamp. We've eventually decided to use the graph properties, otherwise the response output of the get and list commands would be useless:
https://learn.microsoft.com/en-us/graph/add-properties-profilecard#make-additional-attributes-visible-on-the-profile-card

Reply:
They don't work at the moment without changing these values

Reply:
Ah, I see. That's odd!!  @milanholemans, @waldekmastykarz, did you see this? Very odd...

Reply:
For the extension attributes you need to use the Graph property name `customAttribute1` for example. For the other properties you need to use the entra ID name.

Reply:
I've fixed it, but we need to decide what properties to use.